docs/guide/custom-snipptes.md

```diff
@@ -1,7 +1,12 @@
 # Custom Snippets
-Assumed that you need a custom function for your elements, hvstr has the custom snippets feature.
-It allows you to add code to the page-object generation process, which will be added,
-if a given condition is true.
+Assumed that you need hvstr to generate some custom functionality for your elements. Therefore hvstr has the custom snippets feature.
+It allows you to add code to the page-object generation process, which will be added, under a given condition.
+
+There are three different types of custom snippets. Each type will be generated at a different location in the code:
+
+* ### CustomSnippets For Getter functions
+This type of custom snippets will be placed after each getter function of an element. It enables you to add custom function, which for example can toggle an element or selecting an sub element. For this type you will need to generate a function header, because your code will be placed inside the page-object class.
```

Comment:
Hat ein Element mehrere getter? Wird das custom snippet mehrmals generiert?
Falls nein -> will be placed after the getter function...

Reply:
the getterFunction is also an custom Snippet, which is always present.